ouroboros-consensus-shelley/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -158,19 +165,30 @@ data CoreNode c = CoreNode {
     , cnOCert       :: !(SL.OCert               c)
     }
 
+data CoreNodeKeyInfo = CoreNodeKeyInfo
+  { cnkiKeyPair
+      ::  ( SL.KeyPair SL.Regular TPraosMockCrypto
+          , SL.KeyPair SL.Regular TPraosMockCrypto
+          )
+  , cnkiCoreNode :: (SL.KeyPair SL.Genesis TPraosMockCrypto, Gen.AllPoolKeys)
+  }
+
 coreNodeKeys
   :: CoreNode TPraosMockCrypto
-  -> (SL.KeyPair SL.Genesis TPraosMockCrypto, Gen.AllPoolKeys)
-coreNodeKeys CoreNode{cnGenesisKey, cnDelegateKey, cnVRF, cnKES}
-  = ( mkDSIGNKeyPair cnGenesisKey
-    , Gen.AllPoolKeys
-      { Gen.cold = mkDSIGNKeyPair cnDelegateKey
-      , Gen.vrf = mkVRFKeyPair cnVRF
-      , Gen.hot = [(SL.KESPeriod 100, mkKESKeyPair cnKES)]
-      , Gen.hk = SL.hashKey (SL.DiscVKey $ deriveVerKeyDSIGN cnDelegateKey)
+  -> CoreNodeKeyInfo
+coreNodeKeys CoreNode{cnGenesisKey, cnDelegateKey, cnStakingKey, cnVRF, cnKES}
+  = CoreNodeKeyInfo
+      { cnkiCoreNode =
+          ( mkDSIGNKeyPair cnGenesisKey
+          , Gen.AllPoolKeys
+            { Gen.cold = mkDSIGNKeyPair cnDelegateKey
+            , Gen.vrf = mkVRFKeyPair cnVRF
+            , Gen.hot = [(SL.KESPeriod 100, mkKESKeyPair cnKES)]
+            , Gen.hk = SL.hashKey (SL.DiscVKey $ deriveVerKeyDSIGN cnDelegateKey)
```

Comment:
Alignment :grin:

Reply:
Urgh, aligning on identifiers is _such_ a terrible idea. *sigh* If I must :-P

Reply:
Pretty please :slightly_smiling_face:. I understand your point of view, but Edsko and I prefer the improved readability. Moreover, when modifying a module, we should try to adhere to the style used in that module.